be/src/olap/olap_common.h

```diff
@@ -241,7 +246,83 @@ typedef std::set<uint32_t> UniqueIdSet;
 // Column unique Id -> column id map
 typedef std::map<ColumnId, ColumnId> UniqueIdToColumnIdMap;
 
-typedef int64_t RowsetId;
+// 8 bit, id version
+// 120 bit, backend random number generated by uuid
+// 64 bit, inc number from 0
+struct RowsetId {
```

Comment:
Can RowsetId support ++ operator?

Reply:
Can directly use protobuf to do serialize and deserialize?

Reply:
It is useless using ++ , because it will need to get a rowsetidgenerator object. 
Not use pb as a serialize method, we use string.